targetcli/ui_backstore.py

```diff
@@ -611,9 +611,22 @@ def ui_command_create(self, name, size, cfgstring, wwn=None,
 
         try:
             so = UserBackedStorageObject(name, size=size, config=config,
-                                         wwn=wwn, hw_max_sectors=hw_max_sectors)
-        except:
-            raise ExecutionError("UserBackedStorageObject creation failed.")
+                                         wwn=wwn, hw_max_sectors=hw_max_sectors,
+                                         max_data_area_mb=max_data_area_mb)
+        except TypeError as e:
```

Comment:
There is a bug here. I need a generic except statement to catch other exceptions thrown. Will fix and repush.